### PR TITLE
Add README file

### LINKED ISSUE #3: Add a README

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #3

### CODE REVIEW

**Comment by AlmightyYakob** [comment]
This doesn't include the step for copying `.env.default` to `.env`, so it would fail to serve without that.

**Comment by waxlamp** [comment]
> This doesn't include the step for copying `.env.default` to `.env`, so it would fail to serve without that.

Sigh... I will never not remember this. Good catch.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# multinet-docs
+
+This repository contains the source code for the Multinet project documentation.
+Read the live documentation at [Read The
+Docs](https://multinet-app.readthedocs.io), visit the [Multinet project page](),
```
Comment (JackWilb) [inline README.md]: Is there a link for the project page? Is it the VDL one at: https://vdl.sci.utah.edu/projects/2019-nsf-multinet/ ?

Comment (waxlamp) [reply README.md]: Yep, that's the one :joy:

I added the link in f572645bb20cc1db969ccb4984a61877985c5b92.

**Pull request merged**